meta-genivi-dev/recipes-extended/automotive-message-broker/automotive-message-broker_git.bb

```diff
@@ -31,8 +32,16 @@ do_install_append() {
 
     install -d ${D}${systemd_unitdir}/system/
     install -m 0644 ${WORKDIR}/ambd.service ${D}${systemd_unitdir}/system
+    
+    install -d ${D}${libdir}/systemd/user/
+    install -m 0644 ${WORKDIR}/ambd_fsa.service ${D}${libdir}/systemd/user/ambd_fsa.service
+    mkdir -p ${D}/home/root/.config/systemd/user/default.target.wants
+    ln -sf ${libdir}/systemd/user/ambd_fsa.service ${D}/home/root/.config/systemd/user/default.target.wants/ambd_fsa.service.service
+
 }
 
-FILES_${PN} += "${systemd_unitdir}/ambd.service"
+FILES_${PN} += "${systemd_unitdir}/system/ambd.service"
+FILES_${PN} += "${libdir}/systemd/user/*.service \
+                /home/root/.config/systemd/user/default.target.wants/*.service"
```

Comment:
Look at
./meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/gdp-new-hmi.bb,
I took from this example.
ln -sf /usr/lib/systemd/user/gdp-new-hmi.service ${D}/home/root/.config/systemd/user/default.target.wants/gdp-new-hmi.service